### PR TITLE
Smart results

### PR DESCRIPTION
**Pull request by markjmeier**
First draft of perf alert article.  Utilized the current descriptions with some small changes.  Added images and links to sample test results.  Removed divergence alert.  I think we may need to expand on fixes within the CPU/memory alert section

### CODE REVIEW

File: jekyll/_v4/result-analysis/insights-smart-results.md [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+
+### Too Many URLs
+
+This alert is raised when we detect more than 500 unique URLs in your test results. This is commonly caused by a URL that contains a query parameter that is unique per iteration. e.g. tokens, session IDs, etc. You should utilize the URL grouping feature of k6 to combine these unique URLs into a single metrics. You can
+
+*Note:* In some cases, the unique URL may be generated by a third party resource.  As mentioned in the Third Party Content alert, it's a best practice to not include third party resources in your test scripts.
+
+In the following example, our query parameter would produce large number of unique URLs, the name tag would group these all together in our result analysis, making it easier for you to interpret the data.
+
+{% highlight js linenos %}
```
Comment (robingustafsson) [inline jekyll/_v4/result-analysis/insights-smart-results.md]: 👍 good with the example script!

File: jekyll/_v4/result-analysis/insights-smart-results.md [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+          {tags: {name: 'test.loadimpact.com'}});
+{% endhighlight %}
+
+***
+
+### High Load Generator CPU Usage
+
+This alert is raised when we detect high utilization of the load generator CPU during a test. Over utilization of the load generator can skew your test results producing data that varies from test to test and unpredictably. Virtual Users will naturally try to execute as quickly as possible  The exact cause of over utilization can vary, but is likely due to one of the following reasons:
+
+- Lack of sleep times in your scripts
```
Comment (robingustafsson) [inline jekyll/_v4/result-analysis/insights-smart-results.md]: I added another reason to include in this alert to Trello, we could include it here:
> - A JavaScript exception is being thrown early in the VU execution causing an endless restart loop consuming all CPU

File: jekyll/_v4/result-analysis/insights-smart-results.md [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+
+Load Impact's Performance Alerts are algorithms built into Load Impact Insights.  Our algorithms are automatically executed as part of test result processing when you stream your results into our cloud service or execute your test on our cloud infrastructure.  These alerts can be broken down into two catergories Test Health Performance Alerts and Smart Result Performance Alerts.
+
+- TOC
+{:toc}
+
+
+## Smart Result Performance Alerts
+
+Smart Result Performance Alerts are alerts that intelligently analyze your results and will alert you to patterns that we know are related to performance issues on the system under test. Their purpose is to help highlight problem areas so you can spend more time fixing performance problems than analyzing results.
```
[the remark this answers is not in the document]
Comment (markjmeier) [reply jekyll/_v4/result-analysis/insights-smart-results.md]: 😄

File: jekyll/_v4/result-analysis/insights-smart-results.md (hunk 2 above)
Comment (markjmeier) [reply jekyll/_v4/result-analysis/insights-smart-results.md]: Any suggested fixes for this?  Do we/can we provide any insight here?

**Comment by markjmeier** [comment]
@robingustafsson pushed a new commit based on your feedback.  @john-emmitt feel free to review.

File: jekyll/_v4/result-analysis/insights-smart-results.md (hunk 2 above)
Comment (robingustafsson) [reply jekyll/_v4/result-analysis/insights-smart-results.md]: This is one of those cases that we want to detect and notify the user that, the VU iteration failure rate is high. What we could recommend the user to do is try running the script locally with k6 (a validation run) as that will quickly uncover any issue like this.

Comment (markjmeier) [reply jekyll/_v4/result-analysis/insights-smart-results.md]: "Detect and notify".  Sounds like another test health alert to be created?

Comment (robingustafsson) [reply jekyll/_v4/result-analysis/insights-smart-results.md]: Yup, exactly 😄

**Comment by markjmeier** [comment]
Merged. @john-emmitt we can edit the master if you have suggestions.

**Pull request merged**